src/test/java/hw1/tests/MultiplyTest.java

```diff
@@ -0,0 +1,15 @@
+package hw1.tests;
+
+import hw1.dataProviders.MultiplicationDP;
+import org.testng.annotations.Test;
+
+import static org.testng.Assert.assertEquals;
+
+public class MultiplyTest extends AbstractTest {
+    @Test(dataProvider = "Multiplication Data Provider",
```

Comment:
pls add \n (empty string) after class definition to improve readability

Reply:
I modified the code and completed everything as you indicated. Can I commit the revised code for your re-check?